reference/6/Microsoft.PowerShell.Core/About/about_Thread_Jobs.md

```diff
@@ -18,36 +18,59 @@ the current session process.
 
 ## Long description
 
-This article explains how to run thread jobs in PowerShell on a local computer.
-For information about running background jobs on a local computer, see
-[about_Jobs](about_Jobs.md).
+PowerShell concurrently runs commands and scripts through jobs. There are three
+jobs types provided by PowerShell to support concurrency.
 
-Start a thread job by using the `Start-ThreadJob` cmdlet. This cmdlet is
-available in the **ThreadJob** module that ships with PowerShell.
-`Start-ThreadJob` returns a single job object that encapsulates the running
-command or script, and can be used with all PowerShell job manipulating
-cmdlets.
+- `RemoteJob` - Commands and scripts run in a remote session. For information,
+  see [about_Remote_Jobs](about_Remote_Jobs.md).
+- `BackgroundJob` - Commands and scripts run in a separate process on the local
+  machine. For more information, see [about_Jobs](about_Jobs.md).
+- `PSTaskJob` or `ThreadJob` - Commands and script runs in a separate thread
+  within the same process on the local machine.
 
-## The job cmdlets
+Thread based jobs are not as robust as Remote and Background jobs, because they
+run in the same process on different threads. If one job has a critical error
+that crashes the process, then all other jobs in the process are terminated.
+
+However, thread-based jobs require less overhead. They don't use the remoting
+layer or serialization. The result objects are returned as references to live
+objects in the current session. Without this overhead, thread-based jobs run
+faster and use fewer resources than the other job types.
+
+> [!IMPORTANT]
+> The parent session that created the job monitors the job status and collects
```

Comment:
I think "job monitors" looks like a single noun, but it's not intended as such. I'd reword as "The parent session that creates the job also monitors the job status and ..."

Reply:
fixed